sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/planner/BeamRuleSets.java

```diff
@@ -159,14 +158,42 @@
           BeamCoGBKJoinRule.INSTANCE,
           BeamSideInputLookupJoinRule.INSTANCE);
 
+  private static final List<RelOptRule> BEAM_CONVERTERS_CALCITE_SQL_ONLY =
+      ImmutableList.of(BeamCalcRule.INSTANCE);
+
   private static final List<RelOptRule> BEAM_TO_ENUMERABLE =
       ImmutableList.of(BeamEnumerableConverterRule.INSTANCE);
 
   public static RuleSet[] getRuleSets() {
     return new RuleSet[] {
       RuleSets.ofList(
           ImmutableList.<RelOptRule>builder()
-              .addAll(BEAM_CONVERTERS)
+              .addAll(BEAM_CONVERTERS_COMMON)
+              .addAll(BEAM_CONVERTERS_CALCITE_SQL_ONLY)
+              .addAll(BEAM_TO_ENUMERABLE)
+              .addAll(LOGICAL_OPTIMIZATIONS)
+              .build())
+    };
+  }
+
+  /** Returns the rule sets that allow using ZetaSQL evaluator for Calc. */
```

Comment:
All the changes in this file should be reverted. `BeamCalcRule` can be replaced with `BeamZetaSqlCalcRule` in `ZetaSQLQueryPlanner`. https://github.com/apache/beam/blob/4875f4576f6d17ba331ab386d583d99ab4ce1a2d/sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLQueryPlanner.java#L123

Reply:
Done. Thanks for the suggestion! That make the code cleaner.